package.json

```diff
@@ -30,26 +30,15 @@
   "dependencies": {
     "@artsy/express-reloadable": "^1.3.0",
     "@artsy/palette": "^2.19.6",
-    "@artsy/reaction": "^5.3.8",
-    "@babel/cli": "7.0.0-rc.2",
-    "@babel/core": "7.0.0-rc.2",
-    "@babel/node": "7.0.0-rc.2",
-    "@babel/plugin-proposal-class-properties": "7.0.0-rc.2",
-    "@babel/plugin-proposal-decorators": "7.0.0-rc.2",
-    "@babel/plugin-transform-runtime": "7.0.0-rc.2",
-    "@babel/preset-env": "7.0.0-rc.2",
-    "@babel/preset-react": "7.0.0-rc.2",
-    "@babel/preset-typescript": "7.0.0-rc.2",
-    "@babel/register": "7.0.0-rc.2",
-    "@babel/runtime": "7.0.0-rc.2",
```

Comment:
If we're not fully compiling out our server-side CoffeeScript and other ES6 / TypeScript before sending to prod then moving these modules to `devDeps` will fail during deploy. (Things like `coffee-script/register` and `@babel/register` mean that code is transformed during runtime (and then cached).

Reply:
Are we not fully compiling before we deploy? I thought that was what the `yarn assets` step is for.

Reply:
Thats only for the client-side

Reply:
Sounds good, put these back in `dependencies`